### PR TITLE
INDY-1770: decrease iptables clients connections limit

### PR DESCRIPTION
**Pull request by sergey-shilov**
Updated docs, migrated indy.env.

Signed-off-by: Sergey Shilov <sergey.shilov@dsr-company.com>

### CODE REVIEW

File: docs/setup-iptables.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 In order to prevent the indy-node process from reaching of open file descriptors limit caused by clients connections it is strongly
```
Comment (ashcherbakov) [inline docs/setup-iptables.md]: What about `CLIENT_CONNECTIONS_LIMIT=10000` in `postinst_node`?

File: docs/setup-iptables.md [hunk 2]
```diff
@@ -3,6 +3,11 @@
...
 recommended to add iptables rule that limits the number of simultaneous clients connections for client port.
```
Comment (ashcherbakov) [inline docs/setup-iptables.md]: I think a link of this doc and explicit statement about setting firewall rules needs to be part of `README.md` and `start-nodes.md`

File: docs/setup-iptables.md (hunk 1 above)
Comment (sergey-shilov) [reply docs/setup-iptables.md]: Sure.

File: docs/setup-iptables.md (hunk 2 above)
Comment (sergey-shilov) [reply docs/setup-iptables.md]: Updated _setup-iptables.md_ and _start-nodes.md_.

**Pull request merged**